src/hooks/useDownload.ts

```diff
@@ -85,9 +85,9 @@ async function internal<T>({ src, dst, headers, ephemeral, logger }: DownloadOpt
       await body(reader, f, sz)
 
       const text = rsp.headers.get("Last-Modified")
-      const etag = rsp.headers.get("etag")
+      const etag = rsp.headers.get("ETag")
```

Comment:
I think this got in the PR because I force pushed. My bad. I gotta remember you can't do that after a certain point.